### PR TITLE
Balsamic v16 release

### PR DESCRIPTION
**Pull request by ivadym**
## Description

**NOTE**: Do not merge until balsamic release 16.0.0 has been deployed

PR containing all the necessary changes for the Balsamic v16 release.

Changes needed for release 16:
- [x] https://github.com/Clinical-Genomics/cg/issues/3407 **(NOT NECESSARY AS THIS WILL NOT BE ADDED TO RELEASE 16)** 
- [x] update hermes: https://github.com/Clinical-Genomics/hermes/issues/116
- [ ] update order portal: https://github.com/Clinical-Genomics/clinical-genomics-ui/issues/538
- [x] Scout GENS button for TGA: https://github.com/Clinical-Genomics/scout/issues/4884
- [x] Would be nice to add caller info to Scout too: https://github.com/Clinical-Genomics/scout/issues/4901
- [x] Update servers repo with path to version 202308.03 version of Sentieon (https://github.com/Clinical-Genomics/servers/pull/1494)

Optional changes for release 16:
- [x] Artefact DB argument to balsamic: https://github.com/Clinical-Genomics/cg/pull/3792

### Added:
- https://github.com/Clinical-Genomics/cg/pull/3388
- https://github.com/Clinical-Genomics/cg/pull/3361

### Changed
- https://github.com/Clinical-Genomics/cg/pull/3427

### Fixed
-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] workflow integrity tests before start of validation of balsamic has passed:
- [ ] all workflows can be started without error
- [ ] all workflows can be stored, uploaded and delivered without error
 
### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by mathiasbio** [comment]
@khurrammaqbool as the tumor only analysis of MSI will not be added for release 16 I assume that I can remove this  https://github.com/Clinical-Genomics/cg/issues/3407 from the to do list above? : )

**Comment by mathiasbio** [comment]
Code has been tested in individual PRs merged into this. But all aspects will be tested again during the verification tests before starting validation of balsamic. PR will not be merged until all tests there have been successful!

File: cg/meta/workflow/balsamic.py [hunk 1]
```diff
@@ -398,8 +399,12 @@ def get_parsed_observation_file_paths(self, observations: list[str]) -> dict:
 
         return verified_observations
 
-    def get_verified_gens_file_paths(self, sex: Sex) -> dict[str, str] | None:
+    def get_verified_gens_file_paths(self, sex: Sex, panel_bed: str) -> dict[str, str] | None:
```
Comment (ChrOertlin) [inline cg/meta/workflow/balsamic.py:402]: this seems to always return a dict now
```suggestion
    def get_verified_gens_file_paths(self, sex: Sex, panel_bed: str) -> dict[str, str]:
```

**Review by ChrOertlin — APPROVED** [review]

File: cg/meta/workflow/balsamic.py (hunk 1 above)
Comment (mathiasbio) [reply cg/meta/workflow/balsamic.py:402]: True! Nice catch thanks! 🙏

**Review by diitaz93 — APPROVED** [review]
Looks good 🚀

**Pull request merged**